test/sdk-benchmarks/src/main/java/software/amazon/awssdk/benchmark/stats/SdkBenchmarkParams.java

```diff
@@ -46,9 +45,9 @@ public class SdkBenchmarkParams {
 
     private Mode mode;
 
-    @JsonSerialize(using = LocalDateSerializer.class)
```

Comment:
This has been present since 2019. What triggered this change or is it  a refactoring of this field ?

Reply:
Gave a brief explanation in the description

> This is useful when the results are processed across different systems.

It's part of the refactoring work to upload the results to CloudWatch.

Reply:
Thanks 👍